src/components/User/User.tsx

```diff
@@ -37,7 +37,7 @@ export const User = React.forwardRef<HTMLDivElement, UserProps>(
             >
                 {avatar ? (
                     <div className={b('avatar')}>
-                        {React.isValidElement(avatar) ? avatar : <Avatar {...avatar} size={size} />}
+                        {React.isValidElement(avatar) ? avatar : <Avatar size={size} {...avatar} />}
```

Comment:
I suppose it was intentional to put size after avatar props, not allowing to change it

Reply:
got it, placed size prop at the end